front/app/components/FilterSelector/Combobox.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useState, useRef, useEffect, KeyboardEvent } from 'react';
+
+import {
+  Dropdown,
+  colors,
+  fontSizes,
+  isRtl,
+  Box,
+  Button,
+  Icon,
+  useBreakpoint,
+} from '@citizenlab/cl2-component-library';
+import styled from 'styled-components';
+
+import { SelectorProps } from './MultiSelectDropdown';
+import Title from './Title';
+
+import { IFilterSelectorValue } from '.';
+
+const List = styled.ul`
+  margin: 0;
+  padding: 0;
+  list-style: none;
+`;
+
+const ListItemText = styled.span`
```

Comment:
This is 100% duplicated between this component and the `MultiSelectDropdown`, maybe nice to extract to a separate file and reuse